FoxHound.Client/src/Posts/Post.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import Comments, { CommentModel } from "../Comments/Comments";
+import Axios from "axios";
+import { Container, Paper, Box, Typography } from "@material-ui/core";
+import PostContentViewer from "./PostContentViewer";
+import AddComment from "../Comments/AddComment";
+
+export interface PostModel {
+  postId: number;
+  blogId: number;
+  title: string;
+  content: string;
+  createdDate: Date;
+  comments: CommentModel[];
+}
+
+const Post: React.FC = () => {
+  const { postId } = useParams();
+  const [post, setPost] = useState<PostModel>(null);
+  const [error, setError] = useState<string>("");
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+
+  useEffect(() => {
+    getPost();
+  }, []);
+
+  const getPost = async () => {
+    try {
+      setIsLoading(true);
+
+      const response = await Axios.get<PostModel>(
+        `${process.env.API_URL}/Post/Get/${postId}`
+      );
+
+      setPost(response.data);
+    } catch (ex) {
+      setError(ex.message);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  return (
+    <div>
+      {error && <div>{error}</div>}
+
+      {isLoading || !post ? (
+        <div>Loading...</div>
+      ) : (
+        <>
+          <Container maxWidth="lg">
+            <Paper elevation={5}>
+              <Box p={3}>
```

Comment:
Did we ever make a decision whether to use Box of useStyles?

Reply:
I think i'm a fan of the the Box for most simple spacing things. However, I think we need to take a pass or 2 through the UI and fix it up. I'll probably work on that at some point.